### ISSUE TITLE
Go to bug...

### ISSUE THREAD

**Issue by jdw870** [yairm210/Unciv#10926]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Game Version

4.9.18

### Describe the bug

Just go to.... but we can't...

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Screenshots

![Screenshot_20240113_132052_Unciv](https://github.com/yairm210/Unciv/assets/131377953/e4c0b0d7-ff87-43a4-b446-1e2aa8d22f54)
![Screenshot_20240113_132105_Unciv](https://github.com/yairm210/Unciv/assets/131377953/03d96f02-5321-4ef5-a963-7e7d6d83b2be)
![Screenshot_20240113_132043_Unciv](https://github.com/yairm210/Unciv/assets/131377953/af786a3f-e410-4bd7-bb1b-36444c6d3615)

### Link to save file

[그리스 353턴.json](https://github.com/yairm210/Unciv/files/13926757/353.json)

### Operating System

Android

**Comment by tuvus** [comment]
Yes, I have figured out how to modify the code to fix the bug. The bug is about fort building. I was looking to see if I could make the fort building AI functional again, but I think I will just commit a quick fix and work on a more in-depth solution later.
